python/lsst/ts/gis/commander.py

```diff
@@ -31,39 +56,61 @@ def connected(self):
         Returns
         -------
         `bool`
+            Is the client connected?
         """
-        if self.client is None or not self.client.is_socket_open:
+        if self.client is None or not self.client.connected:
             return False
         else:
             return True
 
-    def connect(self):
+    async def connect(self):
         """Connect to the commander."""
```

Comment:
I think it would help a user looking into this code if the doc string were expanded explaining that an `sshtunnel` is used for the connection.